src/app/file-manager/file-upload-modal/file-upload-modal.component.html

```diff
@@ -0,0 +1,20 @@
+<h2 mat-dialog-title>Where do you want to upload to?</h2>
+
+<mat-dialog-content>
+  <div class="flex-container">
+    <mat-form-field class="full-width">
+      <input type="text" matInput placeholder="Select workspace..." [formControl]="workspaceCtrl" [matAutocomplete]="auto">
+      <mat-autocomplete #auto="matAutocomplete" [displayWith]="displayFn.bind(this)" [panelOpen]="false">
+        <mat-option *ngFor="let workspace of filteredWorkspaces | async" [value]="workspace">
+          {{ workspace.name }}
+        </mat-option>
+      </mat-autocomplete>
+    </mat-form-field>
+  </div>
+  <span class="italic">Files will be stored into "uploads" folder</span>
+</mat-dialog-content>
+
+<mat-dialog-actions>
+  <button mat-raised-button (click)="cancel()" class="button-secondary">Cancel</button>
+  <button mat-raised-button (click)="uploadFiles()" class="button-primary">Start Upload!</button>
```

Comment:
There is no `[disabled]="<expression>"` here so I don't see how the button is disabled if no workspace has been selected.

Reply:
Sorry we misunderstood, we thought it was the upload button on the uploads page not on the modal.

Reply:
If it's not easily fixable now I can create a ticket for this.

If nothing is selected, and the user clicks the Upload button, an error is logged in the javascript console when this line is reached:
```
    localStorage.setItem('uploadBucket', this.workspaceCtrl.value.bucketName);
```
in `FileUploadModalComponent.uploadFiles()`